test/parallel/test-fs-stat.js

```diff
@@ -98,5 +98,21 @@ fs.stat(__filename, common.mustCall(function(err, s) {
   console.log('isSymbolicLink: ' + JSON.stringify(s.isSymbolicLink()));
   assert.strictEqual(false, s.isSymbolicLink());
 
+  assert.ok(s.atime instanceof Date);
   assert.ok(s.mtime instanceof Date);
+  assert.ok(s.ctime instanceof Date);
+  assert.ok(s.birthtime instanceof Date);
+}));
+
+fs.stat(__filename, common.mustCall(function(err, s) {
+  const json = JSON.parse(JSON.stringify(s));
+  const keys = [
```

Comment:
On second thought: either put a comment saying why we exclude `blksize`
Or put `if(!common.isWindows) keys.push('blksize')`
Your call but pick one.